run_dir/static/js/sequencing_queues.js

```diff
@@ -29,7 +29,14 @@ function load_table() {
               plate_value = '<div class="mult-pools-margin">'+plate+' <span class="label label-warning sentenceCase">Rerun</span></div>';
             }
             plates.push(plate_value);
-            queuetimes.push('<div class="mult-pools-margin">'+Math.floor(Math.abs(new Date() - new Date(values['queue_time']))/(1000*86400)) + '</div>');
+            var lblinf = '';
+            var wait_time = Math.floor(Math.abs(new Date() - new Date(values['queue_time']))/(1000*86400));
```

Comment:
Maybe add some comments explaining the code here..? 🙄

Reply:
Done!